web/containers/admin/AssessmentPage/index.js

```diff
@@ -18,6 +18,8 @@ class AssessmentPageContainer extends React.Component {
     this.state = {
       applicantType: 'hacker',
       selectedApplicantId: null,
+      sortType: 'timestamp',
+      reverse: false,
```

Comment:
I'm fine with `reverse` but it could be more descriptive if you called it `sortDirection` and allowed the user to pass in either `"asc"` or `"desc"` as the value. It also makes the default state more clear.